code/src/Server.ts

```diff
@@ -428,8 +428,8 @@ try
 
     app.get('/x-nmos/node/v1.3/receivers', function (req, res) {
         res.setHeader('Content-Type', 'application/json');
-        res.send(JSON.stringify(myDevice.FetchReceiversUris()))
-    })
+        res.send(JSON.stringify(myDevice.FetchReceivers()))
```

Comment:
It would be better to pass the jsonIgnoreReplacer as an argument in order not to get properties which should not be in the final serialized JSON response.
```suggestion
        res.send(JSON.stringify(myDevice.FetchReceivers(), jsonIgnoreReplacer));
```

This would mean you need to add an import at the top of the Server.ts file like:

```typescript
import { jsonIgnoreReplacer } from 'json-ignore';
```